cmd/kube-burner/ocp-config/metrics-aggregated.yml

```
@@ -124,6 +124,9 @@
 - query: sum(kube_node_status_condition{status="true"}) by (condition)
   metricName: nodeStatus
 
+- query: count(kube_replicaset_labels{})
+  metricName: replicaSets
```

I think is better to make this query instant and update its name to be consistent with other object counts.

```suggestion
- query: count(kube_replicaset_labels{})
  metricName: replicaSetCount
  instant: true
```